cpp/src/ray/test/cluster/cluster_mode_test.cc

```diff
@@ -276,17 +276,67 @@ TEST(RayClusterModeTest, GetActorTest) {
   EXPECT_FALSE(ray::GetActor<Counter>("not_exist_actor"));
 }
 
-TEST(RayClusterModeTest, CreateAndRemovePlacementGroup) {
+ray::PlacementGroup CreateSimplePlacementGroup(const std::string &name) {
   std::vector<std::unordered_map<std::string, double>> bundles{{{"CPU", 1}}};
 
+  ray::internal::PlacementGroupCreationOptions options{
+      false, name, bundles, ray::internal::PlacementStrategy::PACK};
+  return ray::CreatePlacementGroup(options);
+}
+
+TEST(RayClusterModeTest, CreateAndRemovePlacementGroup) {
+  auto first_placement_group = CreateSimplePlacementGroup("first_placement_group");
+  EXPECT_THROW(CreateSimplePlacementGroup("first_placement_group"),
+               ray::internal::RayException);
+  ray::RemovePlacementGroup(first_placement_group.GetID());
```

Comment:
Can we also use `ray::WaitPlacementGroupReady` to check result here?

Reply:
If creating a same name placement group, the coreworker will return a non-ok status, and then throw an exception, there is no pd_id returned, so can't use WaitPlacementGroupReady(pg_id) here.